test/test_custom_functions.py

```diff
@@ -18,5 +20,8 @@ def test_object_attribute(self):
         self.assertEqual('car.honda.drive(1, 3)', main.main('car.honda.drive(1,3)', 'js', 'py'))
         self.assertEqual('car.ford.park(null)', main.main('car.ford.park(None)', 'py', 'js'))
 
+    #def test_arg_varaible(self):
```

Comment:
Does this work yet?

Reply:
Just added a commit to support this test case @g-harel

Reply:
Is the issue referenced in top level PR [comment](https://github.com/jackdavidweber/cjs_capstone/pull/89#issue-451332354) resolved? If so, maybe edit the comment to reflect it as otherwise it is slightly confusing